executor/hnswpsql.py

```diff
@@ -310,7 +310,8 @@ def search(self, docs: 'DocumentArray', parameters: Dict = None, **kwargs):
             self._vec_indexer.search(docs, parameters)
             kv_parameters = copy.deepcopy(parameters)
             kv_parameters['traversal_paths'] = [
-                path + 'm' for path in kv_parameters.get('traversal_paths', ['r'])
+                path + 'm'
+                for path in kv_parameters.get('traversal_paths', 'r').split(',')
```

Comment:
I think we need to convert this list back to a comma-separated single string. Now it will be for eg. ['rm', 'cm']  but we need 'rm,cm'

Reply:
ah okay, `PostgreSQLStorage` accepts iterable str.

Reply:
no, you are right. it needs to be `','.join`-ed. Done